src/main-config.rs

```diff
@@ -1,10 +1,9 @@
 extern crate cargo_update;
 extern crate tabwriter;
 
-use std::io::{Write, stdout};
-use tabwriter::TabWriter;
+use std::io::{stdout, Write};
 use std::process::exit;
```

Comment:
I'd actually rather this commit dropped, consider the width-priority ordering superiour.

Reply:
Fine, I'll disable `use` sorting in `rustfmt` config.